prow/jobs/jinja/postsubmits/controller_release.jinja2

```diff
@@ -186,10 +186,15 @@
       repo: code-generator
       base_ref: main
       workdir: false
+    - org: aws-controllers-k8s
```

Comment:
Updated the description with:
> This pull request also adds the `ack-chart` repository to the postsubmit job spec, so that `ack-chart` is cloned into the GOPATH of the job.